src/ImageSharp/Advanced/AotCompilerTools.cs

```diff
@@ -7,6 +7,7 @@
 using System.Runtime.CompilerServices;
 using SixLabors.ImageSharp.Formats;
 using SixLabors.ImageSharp.Formats.Bmp;
+using SixLabors.ImageSharp.Formats.Experimental.Tiff;
```

Comment:
Let's not use this `Experimental` namespace. Introducing it created a future breaking change. This format and WebP have great unit tests!

Reply:
Changed with fa6401c0